bin/Powheg/production/2017/13TeV/HJ_MINLO_Pt-200ToInf/HJ_MiNLO_NNPDF31_13TeV.input

```diff
@@ -0,0 +1,67 @@
+numevts NEVENTS
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0      ! energy of beam 1
+ebeam2 6500d0      ! energy of beam 2
+
+hmass 125
+hwidth 0.004221d0
+
+hdecaywidth 0        ! If equals to 1 read total decay width from HDECAY sm.br2 file
+                     ! if 0 the hwidth value is used
+
+
+bwcutoff 15        ! how many W widths below and above the Higgs mass are included
+
+! To be set only if using LHA pdfs
+lhans1   325300      ! pdf set for hadron 1 (LHA numbering)
+lhans2   325300     ! pdf set for hadron 2 (LHA numbering)
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1 ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+
+
+ncall1   150000    ! number of calls for initializing the integration grid
+itmx1    1        ! number of iterations for initializing the integration grid
+ncall2   100000   ! number of calls for computing the integral and finding upper bound
+itmx2    5        ! number of iterations for computing the integral and finding upper bound
+fastbtlbound 1    ! (default 0) if 1 use fast btilde bound
+storemintupb 1    ! store calls for grids
+foldcsi   2       ! number of folds on csi integration
+foldy     2       ! number of folds on  y  integration
+foldphi   2       ! number of folds on phi integration
+nubound 2000    ! number of bbarra calls to setup norm of upper bounding function
```

Comment:
did you test that the settings are high enough to get good results? i see that we used them in the other card i mentioned. however, that doesn't mean much, so i prefer to ask.

Reply:
I started from https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/production/2017/13TeV/Higgs/HJ_MiNLO_NNLOPS_NNPDF31_13TeV/HJ_MiNLO_NNLOPS_NNPDF31_13TeV.input and simply adjusted the `bornktmin`. Then I ran a small test of 5k events with good results for the main quantity of interest
![image](https://user-images.githubusercontent.com/6587412/149994413-22d5cf34-c0e9-46b3-aa11-f7cffae3e540.png)

Reply:
Thanks @nsmith- !